### DISCUSSION TITLE
Is there a way to combine multiple build assets into one?

### DISCUSSION THREAD

**Discussion by nithinkashyapn** [flareact/flareact#161] [Q&A]
Hey,

I want to deploy a blog site built with Flareact on CF workers free plan, but the problem is each page ends up consuming ~8 requests from the daily 100,000 quota. Is there a way to combine these build assets into fewer files so that I can serve a larger community and not worry if a post goes viral?

![multiple](https://user-images.githubusercontent.com/29030918/120952537-e6f0d080-c768-11eb-97a7-4ff6f9dbe981.png)

Thanks!

**Comment by tj-kev** [comment]
This is not something I've tried myself. I think you would run into an issue of the bundle getting bigger over time as you add more pages if you were to combine too many.

**Comment by traffisco** [comment]
You'll get a performance penalty(or warnings) from google if you load huge files.
In theory, you can load the files from a static location and save up on those requests. 
In reality, CF workers are dirt cheap, 5$ for 10m request.  You can serve 1.25m pages for 5$ (assuming 8 requests per page) a month.
Can't think of another host that will serve reliably this volume for this price.

**Reply by tj-kev** [reply]
I would certainly recommend serving images, icons, robots.txt etc from another location like you mentioned. Like S3 and get CF to edge cache them to maintain the performance.
